cmd/convox/env.go

```diff
@@ -282,3 +290,25 @@ func cmdEnvUnset(c *cli.Context) error {
 
 	return nil
 }
+
+func parseEnvLine(line string) (key string, value string, err error) {
+	// Deal with empty lines
+	var m bool
+	if m = regexp.MustCompile(`^\s*$`).MatchString(line); m {
+		return
```

Comment:
nit: Seems the variables declared in the return statement are not used anywhere. To be consistent with the return statements, can we remove the named return variables and `return "","", nil` here? Same with line 303.